### PR TITLE
Fix android rjs v5

### PR DESCRIPTION
**Pull request by blagoev**
fixes RN for Android in RJS v5. 
Realm Object properties not working

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+5.0.3 Release notes (2020-4-01)
+=============================================================
+### Enhancements
+* None.
+
+### Fixed
+* Fixes Realm Object properties not working when accessed, returning undefined on React Native for Android in Realm JS v5.2  (https://github.com/realm/realm-js/issues/2763)
```
Comment (bmunkholm) [inline CHANGELOG.md:7]:
```suggestion
* Fixes Realm Object properties not working when accessed, returning undefined on React Native for Android in Realm JS v5.0.2  ([#2763](https://github.com/realm/realm-js/issues/2763))
```

**Comment by bmunkholm** [comment]
Any tests we can add?

**Comment by blagoev** [comment]
@bmunkholm Not at the moment, after our planned overhaul of the CI it will be.

**Comment by kraenhansen** [comment]
@bmunkholm & @blagoev, if the integration tests pass for React Native on Android the bug is fixed.

**Pull request merged**